src/hooks/data/useVaultApy.ts

```diff
@@ -22,7 +40,24 @@ export function useVaultBaseApr({
 }: VaultDynamicProps) {
   // Preferred: API request
   const apiQuery = useApiVaultDynamic({ poolId, type })
-  // TODO: Fallbacks?
+
+  const vaultAprFallback = useQuery([_address, "vaultAprFallback"], {
+    queryFn: async () => await getVaultAprFallback(_address),
```

Comment:
It's not a big deal but you don't need to define them async like this. You can just do:

```
queryFn: () => getVaultAprFallback(_address)
```